metrics/prometheus.go

```diff
@@ -34,6 +34,13 @@ var (
 		Namespace: "releases",
 	})
 
+	// LastSuccessfulCreate is a timestamp in UTC seconds of the last successful create event
+	LastSuccessfulCreate = prometheus.NewGauge(prometheus.GaugeOpts{
+		Help:      "A timestamp in UTC seconds of the last successful create event",
```

Comment:
We might want to use the word Unix in here (or specify seconds since Jan 1, 1970 00:00:00). UTC is somewhat ambiguous without the epoch)